tests/BioFSharp.Tests.NetCore/BioFSharp/BioCollections.fs

```diff
@@ -127,6 +127,69 @@ let bioCollectionsTests  =
                     "BioArray.translate did not translate the transcript correctly."
             )
 
+            testCase "isEqual" (fun () ->
+                Expect.equal
+                    (testTranscript |> BioArray.isEqual testTranscript)
+                    0
+                    "BioArray.isEqual did not return correct integer when transcripts were equal."
+            )
+
+            testCase "toString" (fun () ->
+                Expect.equal
+                    (aminoAcidSetArray |> BioArray.toString)
+                    "ACDEFGHIKLMNOPQRSTUVWYXJZB-*"
+                    "BioArray.toString did not return the correct string"
+            )
+
+            testCase "toMonoisotopicMass" (fun () ->
+                Expect.floatClose
+                    Accuracy.high
+                    (testProt |> BioArray.toMonoisotopicMass)
+                    // Masses obtained from University of Washington Proteomics Resource https://proteomicsresource.washington.edu/protocols06/masses.php
+                    (131.04048 + 99.06841 + 113.08406)
+                    "BioArray.toMonoisotopicMass did not return correct mass"
+            )
+
+            testCase "toAverageMass" (fun() ->
+                Expect.floatClose
+                    Accuracy.medium // High accuracy was not passing test
+                    (testProt |> BioArray.toAverageMass)
+                    // Masses obtained from University of Washington Proteomics Resource https://proteomicsresource.washington.edu/protocols06/masses.php
+                    (131.19606 + 99.13106 + 113.15764)
+                    "BioArray.toAverageMass did not return correct mass"
+            )
+
+            testCase "toMonoisotopicMassWith" (fun () ->
+                Expect.floatClose
+                    Accuracy.high
+                    (testProt |> BioArray.toMonoisotopicMassWith 18.0) // 18 = mass of one water molecule
+                    // Masses obtained from University of Washington Proteomics Resource https://proteomicsresource.washington.edu/protocols06/masses.php
+                    (131.04048 + 99.06841 + 113.08406 + 18.0)
+                    "BioArray.toMonoisotopicMassWith did not return correct mass"
+            )
+
+            testCase "toAverageMassWith" (fun () ->
+                Expect.floatClose
+                    Accuracy.medium
+                    (testProt |> BioArray.toAverageMassWith 18.0) // 18 = mass of one water molecule
+                    // Masses obtained from University of Washington Proteomics Resource https://proteomicsresource.washington.edu/protocols06/masses.php
+                    (131.19606 + 99.13106 + 113.15764 + 18.0)
+                    "BioArray.toAverageMassWith did not return correct mass"
+            )
+
+            testCase "toCompositionVector" (fun () ->
+                let testCompVec = Array.zeroCreate 26
```

Comment:
In this test you are also dependent on `BioItem.symbol` working as expected. It would make the test more isolated if you initialize the test composition vector without being dependent on other code (e.g. using the correct characters in lines 182-184).

Reply:
To add changes to this PR, the only thing you have to do is add new commits to your branch, they will be automatically added to the PR.

Reply:
Hi @kMutagene,

Thanks for the suggestions. I updated the code to no longer be dependent on BioItem.symbol and committed.

Reply:
Awesome!